cvmfs/cvmfs.cc

```diff
@@ -1900,7 +1903,6 @@ bool Pin(const string &path) {
         return false;
       }
       cache_manager_->Close(fd);
-      return true;
```

Comment:
Maybe coverity called it a bug, but it doesn't change functionality at all as far as I can tell.  Does it?  It just eliminates an unnecessary line of code.